src/utils/calcPoints.js

```diff
@@ -0,0 +1,6 @@
+//@ts-check
+export default function calcPoints(time) {
+  const remain = 30 - time
+  const points = (remain / 3) * 10
+  return Math.floor(points)
+}
```

Comment:
Nice! Maybe you could show those infos as well for the user into the app, so that it´s clearer how many points yout get.

Reply:
Thanks, sooner or later there will be an info page, where you can also find the shortcuts and there i can also write how the points are distributed.